apollo-router/src/services/connector_service.rs

```diff
@@ -167,8 +168,27 @@ async fn execute(
                 http_request: req,
                 context,
             };
+
+            let now = Instant::now();
             let res = client.oneshot(req).await?;
 
+            f64_histogram!(
+                "apollo.router.http.connector.request.duration.seconds",
+                "Duration of connectors HTTP requests",
+                now.elapsed().as_secs_f64(),
+                "subgraph" = original_subgraph_name.clone(),
+                "connector" = connector.id.to_string()
+            );
+
+            u64_counter!(
+                "apollo.router.http.connector.requests.total",
```

Comment:
I don't think we should suffix it by `.total`, prometheus will add `_total` by default